cpp/src/parquet/arrow/path_internal_test.cc

```diff
@@ -34,6 +34,7 @@ namespace parquet::arrow {
 using ::arrow::default_memory_pool;
 using ::arrow::field;
 using ::arrow::fixed_size_list;
+using ::arrow::list_view;
```

Comment:
Why do you need this? You seem to always write `::arrow::list_view` explicitly below?